README.md

```diff
@@ -137,13 +137,13 @@ helm install cloudtty-operator --version 0.4.0 daocloud/cloudtty --set jobTempla
 If cloudtty manages a remote cluster (another cluster than which the cloudtty operator runs on),
 you need tell cloudtty the kube.conf of the remote cluster as below.
 
-You can copy the kube.config `~/.kube/config` from a remote cluster.
+You can copy the kube.config, `~/.kube/config`, from a remote cluster.
```

Comment:
There's only one question, do we need two commas here?

Reply:
Not a must, only an improvement for the two long words with the appositive form. If the change causes any confusion, it can be changed back. That's all right.

Reply:
👌